ai_services/language/python/languageCustomNERSample.py

```diff
@@ -0,0 +1,96 @@
+    compartment_id = compartment_id,
+    training_dataset = oci.ai_language.models.ObjectStorageDataset(dataset_type="OBJECT_STORAGE", location_details=location_details)
+)
+
+print(f"creating model with details:{model_details}")
+model_response = ai_client.create_model(model_details)
+print(f"create_model returned: {model_response.data}")
+
+model_details = ai_client.get_model(model_response.data.id)
+
+#wait till model state becomes ACTIVE
+while (model_details.data.lifecycle_state == "CREATING"):
+    print('Waiting for model creation and training to complete...')
+    time.sleep(1*60) #sleep for 1 minute
+    model_details = ai_client.get_model(model_response.data.id)
+
+print(f"Model status changed from CREATING to {model_details.data.lifecycle_state}")
+
+print("Printing model evaluation results")
+print(model_details.data.evaluation_results)
+
+print("Creating an end point")
+endpoint_details = oci.ai_language.models.CreateEndpointDetails(
+    compartment_id = compartment_id,
+    model_id = model_details.data.id,
+    inference_units = 1,
+    display_name = endpoint_name
+)
+
+endpoint_response = ai_client.create_endpoint(endpoint_details)
+print(f"create_endpoint call returned{endpoint_response.data}")
+end_point_details = ai_client.get_endpoint(endpoint_response.data.id)
+
+#wait till endpoint state becomes ACTIVE
+while (end_point_details.data.lifecycle_state == "CREATING"):
+    print('Waiting for endpoint creation to complete...')
+    time.sleep(1*60) #sleep for 5 minutes
+    end_point_details = ai_client.get_endpoint(end_point_details.data.id)
+
+print(f"End point status changed from CREATING to {end_point_details.data.lifecycle_state}")
+
+text_to_analyze = "\n\nDear Bryan Hernandez,\n \nGilmore  Kennedy and Lloyd is delighted to offer you the position of Chief Strategy Officer with an anticipated start date of 06/16/17, contingent upon background check, drug screening and work permit verification. \n \nYou will report directly to Jeffrey Zamora at Unit 6709 Box 6713,DPO AP 11187. Working hours are decided based on your assigned business unit. \n \nThe starting salary for this position is $216053 per annum. Payment is on a monthly basis by direct deposit done on the last working day of the moth. \n \nGilmore  Kennedy and Lloyd offers a comprehensive benefits program, which includes medical insurance, 401(k), paid time off and gym facilities at work location. \n \nYour employment with Gilmore  Kennedy and Lloyd will be on an at-will basis, which means you and the company are free to terminate employment at any time, with or without cause or advance notice. This letter is not a contract indicating employment terms or duration.\n \nPlease confirm your acceptance of this offer by signing and returning this letter before 7 days from 06/16/17. \n \nSincerely,\n \nCarlos Banks\n(Country Leader, Human Resources)"
+
+enpoint_id = "ocid1.ailanguageendpoint.oc1.iad.amaaaaaaqfzf3paa33qrpn7drx3dmi2mx5feouxlrjhufejdthejzxd4i5gq";
```

Comment:
tou should get this ocid from end_point_details

Reply:
Yes, taking from there only, just forgot to remove this endpoint_id after testing